applications/source/debezium-source/README.adoc

```diff
@@ -104,32 +104,32 @@ Those properties can be used by prefixing them by the `debezium.properties.` pre
-|https://debezium.io/documentation/reference/2.2/connectors/mongodb.html#mongodb-connector-properties
+|https://debezium.io/documentation/reference/2.3/connectors/mongodb.html[MongoDB]
+|https://debezium.io/documentation/reference/2.3/connectors/mongodb.html#mongodb-connector-properties
 
-|https://debezium.io/documentation/reference/2.2/connectors/postgresql.html[PostgreSQL]
-|https://debezium.io/documentation/reference/2.2/connectors/postgresql.html#postgresql-connector-properties
+|https://debezium.io/documentation/reference/2.3/connectors/postgresql.html[PostgreSQL]
+|https://debezium.io/documentation/reference/2.3/connectors/postgresql.html#postgresql-connector-properties
 
-|https://debezium.io/documentation/reference/2.2/connectors/oracle.html[Oracle]
-|https://debezium.io/documentation/reference/2.2/connectors/oracle.html#oracle-connector-properties
+|https://debezium.io/documentation/reference/2.3/connectors/oracle.html[Oracle]
+|https://debezium.io/documentation/reference/2.3/connectors/oracle.html#oracle-connector-properties
 
-|https://debezium.io/documentation/reference/2.2/connectors/sqlserver.html[SQL Server]
-|https://debezium.io/documentation/reference/2.2/connectors/sqlserver.html#sqlserver-connector-properties
+|https://debezium.io/documentation/reference/2.3/connectors/sqlserver.html[SQL Server]
+|https://debezium.io/documentation/reference/2.3/connectors/sqlserver.html#sqlserver-connector-properties
 
-|https://debezium.io/documentation/reference/2.2/connectors/db2.html[DB2]
-|https://debezium.io/documentation/reference/2.2/connectors/db2.html#db2-connector-properties
+|https://debezium.io/documentation/reference/2.3/connectors/db2.html[DB2]
+|https://debezium.io/documentation/reference/2.3/connectors/db2.html#db2-connector-properties
 
-// |https://debezium.io/documentation/reference/2.2/connectors/cassandra.html[Cassandra]
-// |https://debezium.io/documentation/reference/2.2/connectors/cassandra.html#cassandra-connector-properties
+// |https://debezium.io/documentation/reference/2.3/connectors/cassandra.html[Cassandra]
+// |https://debezium.io/documentation/reference/2.3/connectors/cassandra.html#cassandra-connector-properties
 
-|https://debezium.io/documentation/reference/2.2/connectors/vitess.html[Vitess]
-|https://debezium.io/documentation/reference/2.2/connectors/vitess.html#vitess-connector-properties
+|https://debezium.io/documentation/reference/2.3/connectors/vitess.html[Vitess]
+|https://debezium.io/documentation/reference/2.3/connectors/vitess.html#vitess-connector-properties
 
-|https://debezium.io/documentation/reference/2.2/connectors/spanner.html[Spanner]
-|https://debezium.io/documentation/reference/2.2/connectors/spanner.html#spanner-connector-properties
+|https://debezium.io/documentation/reference/2.3/connectors/spanner.html[Spanner]
+|https://debezium.io/documentation/reference/2.3/connectors/spanner.html#spanner-connector-properties
```

Comment:
That's exactly what I meant about support burden if we don't use "versionless" links which essentially points to the latest versions anyway.
Like this one: https://debezium.io/documentation/reference/connectors/spanner.html#spanner-connector-properties

🤷